doc/rados/operations/monitoring.rst

```diff
@@ -626,6 +626,21 @@ For example, the following commands are equivalent to each other:
    ceph daemon osd.0 foo
    ceph daemon /var/run/ceph/ceph-osd.0.asok foo
 
+There are two methods of running admin socket commands: (1)
+using ``ceph daemon`` as described above, which bypasses
+the monitor and assumes a direct login to the daemon's host,
+and (2) using the ``ceph tell {daemon-type}.{id}`` command,
+which is relayed by the monitor and does not require access
+to the daemon's host.
+
+Use the ``raise`` command to send a signal to a daemon, as if by running ``kill -X <daemon.pid>``.
+When run via ``ceph tell`` it allows signalling a daemon without access to its host:
+
+.. prompt:: bash $
+
+   ceph daemon {daemon-name} raise HUP
+   ceph tell {daemon-type}.{id} raise KILL --after 20.5
```

Comment:
Should `--after` be documented explicitly here?

Reply:
I understood that this section's main purpose was to mention the mechanism and give minimal practical examples, encouraging the user to run the help command to learn more.
It didn't feel right to include a full documentation for the raise command while the others were just briefly mentioned.
Hence, the help text for the new command contains all the details